Source/SwiftLintBuiltInRules/Rules/Lint/PrivateSwiftUIStatePropertyRule.swift

```diff
@@ -64,8 +71,27 @@ struct PrivateSwiftUIStatePropertyRule: SwiftSyntaxRule, OptInRule, Configuratio
             }
             """),
             Example("""
+            struct MyApp: App {
+                @StateObject private var model = DataModel()
+            }
+            """),
+            Example("""
+            struct MyScene: Scene {
+                @StateObject private var model = DataModel()
+            }
+            """),
+            Example("""
             struct ContentView: View {
-                @StateObject var foo = Foo()
+                private @StateObject var model = DataModel()
```

Comment:
That's invalid Swift. No need to test it. 😉

Reply:
🤣 Oops. I'll fix it